pyvista/themes.py

```diff
@@ -2197,6 +2209,7 @@ def __repr__(self):
             'Anti-Aliasing': '_anti_aliasing',
             'Split sharp edges': '_split_sharp_edges',
             'Sharp edges feat. angle': '_sharp_edges_feature_angle',
+            'Before close callback': '_before_close_callback',
```

Comment:
If I understand correctly, this value is either `None` or some not very informative repr of a proper function or lambda. Is the purpose of this in the theme repr just to signal whether a callback is set, hoping for a meaningful function name when it is? I guess it wouldn't be easy with the current mechanism to do anything else.

Reply:
I think the purpose is what you mentioned that it helps understand that it is set.  It could also help in debugging.  I'm not sure that it makes sense though.  I mainly implemented as everything else is already included, but this doesn't mean it is a good idea.

A more pressing question is that we serialize the values in some way when saving to file.  This probably fails, and this is untested.  We might need to strip the callback when we save to file.

Reply:
I fixed and added tests for serializing, it now always omits `before_close_callback`.  I'm not sure whether the repr should be modified though.

Reply:
I think we should keep it for now. It will be None in most cases and it might be informative in other cases. If for some reason it ends up being weird we can change or remove it later.